x-pack/legacy/plugins/uptime/public/state/reducers/overview_filters.ts

```diff
@@ -49,6 +49,7 @@ export function overviewFiltersReducer(
       return {
         ...state,
         errors: [...state.errors, action.payload],
+        loading: false,
```

Comment:
Why did this change go in this PR? Is it fixing a separate bug?

Reply:
yes, it wasn't setting loading false if api was failing.